rajawali/src/main/java/c/org/rajawali3d/engine/EngineImpl.java

```diff
@@ -35,76 +33,90 @@
  * @author Jared Woolston (Jared.Woolston@gmail.com)
  */
 @NotThreadSafe
-public class RendererImpl implements Renderer, ISurfaceRenderer {
+public class EngineImpl implements Engine, ISurfaceRenderer {
```

Comment:
I like that idea.  I would think we can abstract out any shared implementations (e.g. frame timing/management?) if/when we add a second render API